.gitlab/os.yml

```diff
@@ -4,7 +4,7 @@
 .sys_config:
   variables:
     LLNL_SERVICE_USER: sphapp
-    UPSTREAM_DIR: /usr/WS2/wciuser/Spheral/spheral-spack-tpls
+    UPSTREAM_DIR: /usr/WS2/sduser/Spheral/0.22
```

Comment:
Can we name the directory something a little more descriptive than 0.22 so we know its the spack tpl upstream directory.

Reply:
Yeah let me look into how I can do that without having to reinstall everything in the upstream. It should be doable.